src/test/java/org/openelisglobal/integrationTests/patientIdentityType/PatientIdentityTypeServiceTest.java

```diff
@@ -0,0 +1,85 @@
+package org.openelisglobal.integrationTests.patientIdentityType;
+import static org.junit.Assert.*;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openelisglobal.BaseTestConfig;
+import org.openelisglobal.common.exception.LIMSDuplicateRecordException;
+import org.openelisglobal.config.TestConfig;
+import org.openelisglobal.patientidentitytype.service.PatientIdentityTypeService;
+import org.openelisglobal.patientidentitytype.valueholder.PatientIdentityType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
```

Comment:
i would add space between the imports and the annotations

Reply:
sure @mherman22